src/store/actions/recommendationActions.js

```diff
@@ -1,18 +1,28 @@
-import axiosWithAuth from "../../utils/axiosWithAuth.js";
+import axios from "axios";
 export const FETCHING_RECOMMENDATIONS_START = "FETCHING_RECOMMENDATIONS_START";
 export const FETCHING_RECOMMENDATIONS_SUCCESS =
   "FETCHING_RECOMMENDATIONS_SUCCESS";
 export const FETCHING_RECOMMENDATIONS_FAIL = "FETCHING_RECOMMENDATIONS_FAIL";
 
 // RECOMMENDATIONS
 // this call requests new recommendations be inserted into the database, then returns latest
-export function recommendationAction(id, accessToken) {
+export function recommendationAction(id) {
   return (dispatch) => {
     dispatch({
       type: FETCHING_RECOMMENDATIONS_START,
     });
-    axiosWithAuth(accessToken)
-      .get(`/${id}/recommendations`)
+    axios
+      .post(
+        "https://ds.groa.us/recommendations",
+        {
+          user_id: id,
+          num_recs: 50,
```

Comment:
Should this number be 25?

Reply:
I think it's 25 for explore page and 50 for recommendations page.